### ISSUE TITLE
Initialization fails after running out of disk space

### ISSUE THREAD

**Issue by link2xt** [khoj-ai/khoj#369]
I have configured some folder with org files for search and khoj process started to download models from huggingface.
Now this partially downloaded model causes error messages:
```
                    OSError: .../.khoj/search/image/sentence-transformers_clip-ViT-B-32 does not appear to have a file named config.json.
```

Web UI says "Unable to reinitialize. Raise issue on Khoj Github or Discord.".

I will remove .khoj and restart from scratch.

**Comment by sabaimran** [comment]
Interesting -- were you unblocked if you cleared some of your disk space?

Agreed that's not a pleasant experience. Ideally it would rollback partial writes if it failed in the middle, but this might be difficult if it just ends up hanging in the middle.

**Comment by sabaimran** [comment]
That's a wonderful suggestion. That will definitely be helpful for our codepath that downloads llamav2 as well -- it's a massive 3.6GB file with some propensity to failure.

**Comment by debanjum** [comment]
This was resolved in #393
